### ISSUE TITLE
brandlogo url relative to where compiled

### ISSUE THREAD

**Issue by kcrisman** [rbeezer/mathbook#114]
It seems that in `<docinfo>`, the <brandlogo source="...">` attribute is called relative to the directory one compiles in, not to the location of the xml file where the `<docinfo>` occurs.

That's fine, but if so then it should be documented, since one probably usually will NOT run `xsltproc` in the same directory as the source in order to avoid cluttering up the source directory with the generated files.  Otherwise perhaps this behavior should be changed.

**Comment by rbeezer** [comment]
This attribute/URL, like for all images, is relative to wherever you host the pages containing the images.  Eventually, the HTML will live on a webserver and you should anticipate that scenario.  Until then, it is your choice for the local destination of content pages, and directories of images.

Note there is a configurable directory prefix (which maybe I like and maybe I don't).

Short answer: image management is not 100% automatic.  Writing some project management scripts early is always a good idea.

I'm open to a suggestion for where to explain this?

**Comment by Alex-Jordan** [comment]
To close this, I propose someone write a section of the documentation guide that is all about the various things that can go into a `docinfo` element. Including a `brandlogo`, which could discuss the issue here.

**Comment by kcrisman** [comment]
> To close this, I propose someone write a section of the documentation
> guide that is all about the various things that can go into a docinfo
> element. Including a brandlogo, which could discuss the issue here.
>
>
Yes, please!  A lot of us who have older projects not using the full
capabilities would benefit from this too, and new projects can then get it
right from the start.

>

**Comment by rbeezer** [comment]
Significant parts of docinfo should migrate to the publisher file.

**Comment by davidfarmer** [comment]
@kcrisman I think the suggestion was that you write it.

**Comment by kcrisman** [comment]
>
> @kcrisman I think the suggestion was that you write it.
>
>
>
You are right, I had forgotten that - only saw the replies!  Unfortunately
in that case it will be some time before it happens :(
